x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/routes.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Route } from 'react-router-dom';
+// import { PLUGIN } from '../common/constants';
+// import { Detail, DetailProps } from './screens/detail';
+import { Home } from './screens/home';
+
+// patterns are used by React Router and are relative to `APP_ROOT`
+// XXX does nesting react routers like this even work?
```

Comment:
Yep, react routers can be nested, and hopefully this file can be removed in favor of just having all the routes in `index.tsx` (I see that already has `Home` set up!)